runtime/onert/backend/cpu/ops/LogisticLayer.cc

```diff
@@ -42,14 +61,29 @@ void LogisticLayer::logisticFloat32()
 
 void LogisticLayer::logisticQuant8()
 {
-  // cker quant8 logistic is not implemented yet
-  throw std::runtime_error{"NYI"};
+  const int size = MatchingFlatSize(getTensorShape(_input), getTensorShape(_output));
+  const uint8_t *input_data = reinterpret_cast<const uint8_t *>(_input->buffer());
+  uint8_t *output_data = reinterpret_cast<uint8_t *>(_output->buffer());
+
+  for (int i = 0; i < size; ++i)
+  {
+    output_data[i] = _table[input_data[i]];
```

Comment:
I put output_data here instead of invoking cker function because there is already the lookup table.